### PR TITLE
Added argument to save surefire log txt to artifact zip if test runs without error; fixed python test run for grind

### PR DESCRIPTION
**Pull request by bgaborg**

### CODE REVIEW

File: grind/bin/grind [hunk 1]
```diff
@@ -398,6 +402,9 @@ class TestRunner:
         java_version = self.args.java_version or self.project_config.java_version
         assert java_version in supported_java_versions
 
+        if self.args.store_txt_output is True:
+         self.project_config.artifact_archive_globs = ["**/surefire-reports/TEST-*.xml", "**/surefire-reports/*.txt"]
```
Comment (umbrant) [inline grind/bin/grind]: Indentation looks off here.

Also, it looks like this overrides the project_config globs? Shouldn't it append instead?

Comment (bgaborg) [reply grind/bin/grind]: Yes, you are right, I'll fix that. It should append. 
I don't know if there is a bug in the indentation, I'll check my settings in pycharm, using defaults.

**Comment by bgaborg** [comment]
I've done the requested changes. 
Could you point me where to set the globs to include txt files in artifact zip IF all test runs successfully? If there is another place to set this, and it's more convenient then we do not need this feature indeed.

**Comment by bgaborg** [comment]
Okay, so we've found out that per project settings could be used for this as described in https://github.com/cloudera/dist_test/blob/master/docs/grind.md#per-project-configuration. Although it could be a neat feature that this could be used as an argument.

**Comment by bgaborg** [comment]
Closing because the output in the txt file will not solve our problem.

**Pull request closed without merging**